### PR TITLE
replace hipMemcpy with hipMemcpyWithStream

### PR DESCRIPTION
**Pull request by fsx950223**

### CODE REVIEW

**Review by carlushuang — APPROVED** [review]

**Comment by illsilin** [comment]
Hey guys, looks like this PR broke the develop branch:
http://micimaster.amd.com/blue/organizations/jenkins/MLLibs%2Fcomposable_kernel/detail/develop/984/pipeline

Please always run the clang-format script before you push any changes to github and wait for CI to pass before merging into develop.

**Pull request merged**
